addons/api/addon/models/auth-method.js

```diff
@@ -118,9 +117,11 @@ export default class AuthMethodModel extends GeneratedAuthMethodModel {
    */
   changeState(state, options = { adapterOptions: {} }) {
     const defaultAdapterOptions = {
-      method: 'change-state',
       state,
     };
+    if (this.isOIDC) {
```

Comment:
why is this only for OIDC?

Reply:
great question, only oidc has a 'change-state' endpoint. For ldap auth-methods, specifying a method causes the API to return an error. 
![image](https://github.com/hashicorp/boundary-ui/assets/107949262/49bff413-82f2-4010-a7bd-9655462b2682)

Reply:
So we can't change the LDAP state after creation?

Reply:
Ldap state can be updated, its just that specifying the 'change-state' method for ldap will cause an error. The `serializeWithState` method in the serializer takes care of sending a payload with just the state.